File: Br.Com.Fiap.Gere.Residuo/Model/UsuarioModel.cs
namespace Br.Com.Fiap.Gere.Residuo.Model
{
    public class UsuarioModel
    {
        public int UsuarioId { get; set; }
        public string UsuarioNome { get; set; }

        public string UsuarioEmail { get; set; }

        public string UsuarioSenha { get; set; }

        public UsuarioRole UsuarioRole { get; set; }
    }
}

File: BDD_Tests/BDD-Gere-Residuo/src/test/java/model/usuario/UsuarioModel.java
package model.usuario;
import com.google.gson.annotations.Expose;
import lombok.Data;

@Data
public class UsuarioModel {

    @Expose(serialize = false)
    private int usuarioId;

    @Expose
    private String usuarioNome;

    @Expose
    private String usuarioEmail;

    @Expose(deserialize = false)
    private String usuarioSenha;

    @Expose
    private UsuarioRole usuarioRole;
}
